src/definitions.js

```diff
@@ -13,6 +13,10 @@ export const SCRAPERS = {
     name: 'Discount Bank',
     loginFields: ['id', PASSWORD_FIELD, 'num'],
   },
+  otsarHahayal: {
+    name: 'Bank Otsar Hahayal',
+    loginFields: ['userCode', PASSWORD_FIELD],
```

Comment:
@matanelgabsi `userCode` here should also be renamed to `username`

Reply:
Ohh sorry about that. Will commit a fix in a few minutes